src/helpers/clipboard/clipboard_figure_state.ts

```diff
@@ -84,14 +84,19 @@ export class ClipboardFigureState implements ClipboardState {
    */
   paste(target: Zone[]) {
     const sheetId = this.getters.getActiveSheetId();
+    const { width, height } = this.copiedFigure;
+    const numCols = this.getters.getNumberCols(sheetId);
+    const numRows = this.getters.getNumberRows(sheetId);
+    const { start: colStart } = this.getters.getColDimensions(sheetId, target[0].left);
```

Comment:
```suggestion
    const targetX = this.getters.getColDimensions(sheetId, target[0].left).start;
```

Kinda nitpicking, `colStart` isn't a great variable name since we don't know which col we're talking about +  unpacking is useless (and harder to read) when you only need az single variable.

Same comment for the 4 lines below